hadoop-ozone/integration-test/src/test/java/org/apache/hadoop/ozone/scm/node/TestDecommissionAndMaintenance.java

```diff
@@ -283,28 +279,13 @@ public void testStoppedDecommissionedNodeTakesSCMStateOnRestart()
     waitForDnToReachPersistedOpState(dn, IN_SERVICE);
   }
 
-  @Test
-  // A node which is decommissioning or decommissioned can be move back to
-  // IN_SERVICE.
-  public void testDecommissionedNodeCanBeRecommissioned() throws Exception {
```

Comment:
This test has been effectively merged into testNodeWithOpenPipelineCanBeDecommissioned by recommissioning the node after it completes decommission.